package.json

```diff
@@ -22,6 +22,6 @@
   },
   "engines": {
     "vscode": "*",
-    "sqlops": "*"
+    "sqlops": "feat/serilizeConnectionString"
```

Comment:
This is just for development and you're going to set it back to "*" before merging, right?